install/sensors.csv

```diff
@@ -854,7 +854,7 @@ Nanometrics,Trillium Compact 120PH-2,9335,MNTZ,10,0,,0,1.2,0,0,0,0,2022-12-19T22
 Nanometrics,Trillium Compact 120PH-2,9336,PAKZ,10,0,,0,1.2,0,0,0,0,2022-12-19T00:05:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9337,PTAZ,10,0,,0,1.2,0,0,0,0,2022-12-08T21:30:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9365,HTPZ,10,0,,0,1.2,0,0,0,0,2022-12-19T02:31:00Z,9999-01-01T00:00:00Z
-Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,12,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,9999-01-01T00:00:00Z
+Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,12,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,2023-12-13T22:03:00Z
```

Comment:
Sorry I think this is the wrong sensor. It should be a `20PH-2` (20s models rather than 120s)